docs/api_migration_plan.md

```diff
@@ -82,3 +82,4 @@ All non-successful responses should return appropriate status codes in the range
 | `POST   /cbs/login` | | `{ query: { email, password } }` | `{ result: { cb_admin_token } }` | `POST /admin/login` |
 | `POST   /cbs/:id/emails` | | `{ query: { password_reset: Bool } }` | `{ result: null }` | `POST /cb/pwd/reset` |
 | `POST   /visitors/statistics` | `Authorization: cb_admin_token` | `{ query: { group_by: { gender, activity, age: [18, 35, 51, 70], days: ['monday'] } }` | `{ result: { gender: { male, female, ... }, activity: { yoga: 12, ...}, age: { 0-17: 2, ... }, days: { monday: 2, ... } } }` | `POST /users/chart-all` <br/> `POST /users/filtered` |
+| `POST   /cb/feedback` | `Authorization: cb_admin_token` | ` { query: { feedbackScore }` | `[ { result: { cbEmail, feedbackScore, feedback_date } } ]`| ✨ shiny shiny new ✨ |
```

Comment:
This is still inaccurate:

<img width="265" alt="screen shot 2018-03-15 at 10 57 00" src="https://user-images.githubusercontent.com/12719553/37459332-a28fe622-283f-11e8-8567-742acb36a342.png">